puppyhoffers/mac.sh

```diff
@@ -869,8 +869,16 @@ xGB1zXS6cgjtMdz/AIVUdzKo9C7tGM+1R8U7diPFQFq1RydT/9kK
 osascript <<-===
   tell application "System Events"
     set theDesktops to a reference to every desktop
-    repeat with x from 1 to (count theDesktops)
-      set picture of item x of the theDesktops to "~/puppyhoff.jpg"
+    set spaceCount to 10
+
+    repeat with space from 1 to spaceCount
+      set picture rotation of thisDesktop to 0
```

Comment:
this line fails when i test this out
```200:201: execution error: The variable thisDesktop is not defined. (-2753)```
maybe if i'd tried it when you submitted it 2.5 years ago it would have worked better...